webpack.config.js

```diff
@@ -152,6 +152,7 @@ module.exports = {
                   path.resolve(nodedir, "bootstrap-sass", "assets", "stylesheets"),
                 ],
                 outputStyle: "compressed",
+                quietDeps: true,
```

Comment:
I don't think this will take effect, you need 1.35.0 sass version, and I did not check for the sass-loader https://github.com/sass/dart-sass/blob/main/CHANGELOG.md#1350

Reply:
I tried it and it worked :shrug:

Reply:
the version for sass, btw:

```
  "_from": "sass@^1.27.1",
  "_id": "sass@1.35.1",
```

Because we use a `^` we indeed get 1.35.